cinema/serializers.py

```diff
@@ -68,13 +84,64 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSessionSerializer(serializers.ModelSerializer):
```

Comment:
TicketDelailSerializer?

Reply:
Move all Ticket serializers in one place, so they will be one by one, could be easier